### PR TITLE
add provided but unused ruby icon

### PR DESCRIPTION
**Pull request by CantoroMC**

### CODE REVIEW

File: src/icons-nerdfont.h [hunk 1]
```diff
@@ -210,6 +210,7 @@
 #define ICON_EXT_RPM       ICON_ARCHIVE
 #define ICON_EXT_RSS       "\uf96b"
 #define ICON_EXT_RTF       "\uf724"
+#define ICON_EXT_RUBY      ICON_RUBY
```
Comment (luukvbaal) [inline src/icons-nerdfont.h:213]: This should be named ICON_EXT_RB.

Comment (CantoroMC) [reply src/icons-nerdfont.h]: @luukvbaal corrected

**Comment by luukvbaal** [comment]
LGTM

**Comment by jarun** [comment]
Thank you!

**Pull request merged**